vector/src/main/java/im/vector/app/features/location/LocationSharingService.kt

```diff
@@ -143,6 +142,17 @@ class LocationSharingService : VectorService(), LocationTracker.Callback {
 
     override fun onLocationUpdate(locationData: LocationData) {
         Timber.i("### LocationSharingService.onLocationUpdate. Uncertainty: ${locationData.uncertainty}")
+        roomArgsList.forEach { roomArg ->
+            val room = activeSessionHolder.getSafeActiveSession()?.getRoom(roomArg.roomId)
+            room?.getStateEvent(EventType.STATE_ROOM_BEACON_INFO.first())?.let { beaconInfoEvent ->
+                room.sendLiveLocation(
+                        beaconInfoEventId = beaconInfoEvent.eventId!!,
```

Comment:
Nit: Would it make sense to chain this before the `let` to avoid the force unwrap? Looks like you're not reading anything else from the event here anyway.

Reply:
Nice catch, done. (Although I don't know why event id is nullable).